student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+
+class Student < Person
+    def initialize (classroom)
+        @classroom = classroom
+    end
```

Comment:
- [ ] As stated in the project requirement when you implemented  Constructor extends parent's constructor by adding @classroom and a parameter for it. Kindly I recommend you to use `Super `  in the constructor of a subclass to extend and call the constructor of its parent class. This ensures that the initialization code from the parent class is executed before the specific initialization code of the subclass Just like below sample.

```
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
fixed